compiler/tfldump/src/Dump.cpp

```diff
@@ -382,6 +383,14 @@ void dump_model(std::ostream &os, const tflite::Model *model)
   }
   os << std::endl;
 
+  // dump metadata
+  os << "metadata : B(index) name" << std::endl;
```

Comment:
Maybe surround with `if (metadata != nullptr) {` ?

Reply:
I've checked with old inception_v3_2018_04_27.tgz which doesn't seem to have metadata; _tfldump_ just exits without any message.